### ISSUE TITLE
Configuration service :  possible activate operation limitation

### ISSUE THREAD

**Issue by BEHAL** [SamCooper/COMMON_SPEC_RIDS#61]
Can an activate operation activate a new configuration of the provider that is currently running this operation ?

Please clarify, and if it is not possible, complete the activate operation description.

**Comment by SamCooper** [comment]
I don't understand your question, can you rephrase it?

**Comment by BEHAL** [comment]
When you use the activate operation, you use a service provided by a provider. If you need to activate a new configuration for this provider, can you use the configuration service of this provider to do it or do you have to use the configuration service of another provider ? In other words, if I want to change the service configuration of Provider A, can I use the configuration service of provider A to do so ? (It is how windows works).

**Comment by SamCooper** [comment]
Either model is supported, the location of the Configuration service is deployment specific.
